sensitivity.tex

```diff
@@ -108,63 +116,51 @@ \subsection{Lifetime Extension of French \glspl{LWR}}\label{sec:life}
-This is balanced by extending the necessary duration of 
-\gls{LWR} \gls{UNF} reprocessing,
-so
-the total amount of \gls{LWR} \gls{UNF} reprocessed increases nonetheless.
-
-
-The differential impacts of varying the French \gls{LWR} lifetimes are
-shown in table \ref{tab:ext_diff}. The differences were calculated
-using the same equation used for the breeding ratio sensitivity study.
 
+Figure \ref{fig:ext_fuel} shows the change in \gls{ASTRID} fuel loading with
+\gls{LWR} lifetime extension. The \gls{ASTRID} fuel loaded with plutonium
+from \gls{LWR} \gls{UNF} convey the corresponding \gls{LWR} \gls{UNF} reprocessing demand.
+The change in \gls{ASTRID} deployment alters \gls{ASTRID} fuel loading patterns,
+but does not increase the \gls{LWR} \gls{UNF} demand significantly.
 
-\begin{figure}[htbp!]
-	\begin{center}
-		\includegraphics[scale=0.7]{./images/sensitivity/ext_uox_rep.png}
-	\end{center}
-	\caption{Increasing the lifetime of French \glspl{LWR} decreases the monthly
-		\gls{UOX} reprocessing demand, but increases the duration of \gls{LWR} \gls{UNF}
-		reprocessing, thereby increasing the total amount of \gls{LWR} \gls{UNF} reprocessed.}
-	\label{fig:ext_uox}
+\begin{figure}[!ht]
+	\centering
+	\subfloat{\includegraphics[width=.45\textwidth]{./images/sensitivity/base.png}}\quad
+	\subfloat{\includegraphics[width=.45\textwidth]{./images/sensitivity/5yr.png}}\\
+	\subfloat{\includegraphics[width=.45\textwidth]{./images/sensitivity/10yr.png}}\quad
+	\subfloat{\includegraphics[width=.45\textwidth]{./images/sensitivity/20yr.png}}
+	\caption{\gls{ASTRID} fuel loading patters are altered by changes in \gls{ASTRID} deployment
+			 caused by the lifetime extension of \glspl{LWR}.}
+	\label{fig:ext_fuel}
 \end{figure}
 
-Figure \ref{fig:ext_all} shows that lifetime extension has little
-effect on the \emph{average} monthly reprocessing demand, because
-the amount of plutonium in the \gls{ASTRID} used fuel remains the same.
-The delay of \gls{ASTRID} deployment simply provides more time for
-France to prepare the fuel. 
-Extended \gls{LWR} lifetimes delay transition, and provide France with
-time to accumulate \gls{LWR} \gls{UNF}. However, these 
-license extensions do not reduce
-the total reprocessing capacity required for transition.
-
-
-\begin{figure}[htbp!]
-    \begin{center}
-        \includegraphics[scale=0.7]{./images/sensitivity/ext_tot_rep.png}
-    \end{center}
-    \caption{Increasing the lifetime of French \glspl{LWR} simply delays the
-             reprocessing demand, and has little impact on the total 
-     reprocessing capacity required.}
-    \label{fig:ext_all}
-\end{figure}
+The quantitative effects of
+\gls{LWR} lifetime extensions are shown in table \ref{tab:ext_met}.
+The differences were calculated using the same equation used for the
+breeding ratio study. Different metrics were studied for the lifetime
+extension study since there are less \gls{ASTRID} fuel produced due to
```

Comment:
```suggestion
Since \gls{LWR} lifetime extensions shorten the span of \gls{ASTRID} operations, less \gls{ASTRID} fuel is needed when \gls{LWR} lifetimes are extended.
```